snippets/Google.Datastore.V1Beta3.Snippets/DatastoreClientSnippets.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using System.Linq;
+using Xunit;
+using static Google.Datastore.V1Beta3.CommitRequest.Types;
+using static Google.Datastore.V1Beta3.Key.Types;
+using static Google.Datastore.V1Beta3.PropertyFilter.Types;
+using static Google.Datastore.V1Beta3.ReadOptions.Types;
+
+namespace Google.Datastore.V1Beta3.Snippets
+{
+    [Collection(nameof(DatastoreSnippetFixture))]
+    public class DatastoreClientSnippets
+    {
+        private readonly DatastoreSnippetFixture _fixture;
+
+        public DatastoreClientSnippets(DatastoreSnippetFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void Lookup()
+        {
+            var projectId = _fixture.ProjectId;
+            var key1 = _fixture.PrideAndPrejudiceKey;
+            var key2 = new Key
+            {
+                PartitionId = _fixture.PartitionId,
+                // Likely not to be found...
+                Path = { new PathElement { Id = key1.Path[0].Id - 1, Kind = _fixture.BookKind } }
+            };
+
+            // <Lookup>
+            var client = DatastoreClient.Create();
+            var response = client.Lookup(
+                projectId,
+                new ReadOptions { ReadConsistency = ReadConsistency.STRONG },
+                new[] { key1, key2 });
+            Console.WriteLine($"Found: {response.Found.Count}");
+            Console.WriteLine($"Deferred: {response.Deferred.Count}");
+            Console.WriteLine($"Missing: {response.Missing.Count}");
+            // </Lookup>
+
+            var entity = response.Found[0].Entity;
+            Assert.Equal("Jane Austen", (string)entity["author"]);
+            Assert.Equal("Pride and Prejudice", (string)entity["title"]);
+        }
+
+        [Fact]
+        public void StructuredQuery()
+        {
+            var projectId = _fixture.ProjectId;
+            var partitionId = _fixture.PartitionId;
+            var kind = _fixture.BookKind;
```

Comment:
I think setting up kind and partitions are one of the reasons that people would read this snippet